Makefile

```diff
@@ -0,0 +1,14 @@
+default: clean depsupdate lint
```

Comment:
You might have meant for `depsupdate` here to just be `deps`, so that a simple `make` command doesn't update composer dependencies unexpectedly. Sorry to mention that after this has been merged.

Reply:
OK, I fixed it in develop.